tests/lean/io_bug2.lean.expected.out

```diff
@@ -1,3 +1,3 @@
+
 "t1"
 "t2"
```

Comment:
Note: these are due to the `#eval` result (unit) being printed before trace messages.